<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StateMachineWin</class>
 <widget class="QWidget" name="StateMachineWin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>194</y>
     <width>53</width>
     <height>37</height>
    </rect>
   </property>
   <property name="text">
    <string>NONE</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>92</x>
     <y>164</y>
     <width>409</width>
     <height>93</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QPushButton" name="btn1">
      <property name="text">
       <string>BTN1</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="btn2">
      <property name="text">
       <string>BTN2</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="btn3">
      <property name="text">
       <string>BTN3</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_1">
   <property name="geometry">
    <rect>
     <x>98</x>
     <y>298</y>
     <width>53</width>
     <height>37</height>
    </rect>
   </property>
   <property name="text">
    <string>MOVE</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>304</x>
     <y>296</y>
     <width>53</width>
     <height>37</height>
    </rect>
   </property>
   <property name="text">
    <string>NOT MOVE</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>98</x>
     <y>384</y>
     <width>53</width>
     <height>37</height>
    </rect>
   </property>
   <property name="text">
    <string>DIRTY</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>384</y>
     <width>53</width>
     <height>37</height>
    </rect>
   </property>
   <property name="text">
    <string>CLEAN</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_started">
   <property name="geometry">
    <rect>
     <x>158</x>
     <y>282</y>
     <width>131</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;-- Started &lt;--</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_stopped">
   <property name="geometry">
    <rect>
     <x>158</x>
     <y>318</y>
     <width>129</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>--&gt; Stopped --&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_soiled">
   <property name="geometry">
    <rect>
     <x>162</x>
     <y>370</y>
     <width>125</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;-- Soiled &lt;--</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_cleaned">
   <property name="geometry">
    <rect>
     <x>162</x>
     <y>406</y>
     <width>125</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>--&gt; Cleaned --&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_carMove">
   <property name="geometry">
    <rect>
     <x>378</x>
     <y>296</y>
     <width>185</width>
     <height>37</height>
    </rect>
   </property>
   <property name="text">
    <string>None</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_carAir">
   <property name="geometry">
    <rect>
     <x>382</x>
     <y>382</y>
     <width>185</width>
     <height>37</height>
    </rect>
   </property>
   <property name="text">
    <string>None</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
